marimo/_messaging/ops.py

```diff
@@ -468,6 +469,14 @@ def __init__(
 
     def _stringify(self, value: object) -> str:
         try:
+            # HACK: We pretty-print tables to avoid str(ibis_table)
+            # which can be very slow when `ibis.options.interactive = True`
+            table_manager = get_table_manager_or_none(value)
+            if table_manager:
```

Comment:
Good to explicitly check for `None` as a rule. I picked up this habit [at Google](https://google.github.io/styleguide/pyguide.html#2144-decision) and it helps avoid bugs with `False`y objects.

```suggestion
            if table_manager is not None:
```